internal/kafka/test/integration/kafkas_test.go

```diff
@@ -1189,10 +1192,81 @@ func TestKafkaDenyList_UnauthorizedValidation(t *testing.T) {
 		},
 	}
 
+	RegisterTestingT(t)
+
+	// verify that the user does not have access to the service
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			resp := tt.operation()
```

Comment:
Thanks @ziccardi :) I not sure if I'm misunderstanding this but from your example would you suggest:
```
	for _, tt := range tests {
		t.Run(tt.name, func(t *testing.T) {
			temp := tt
			resp := (&temp).operation()
```

Reply:
No, because you are still referencing `tt` from inside the function.
A working code would be:
```
for _, tt := range tests {
		temp := tt
		t.Run(tt.name, func(t *testing.T) {
			resp := (&temp).operation()
```

however, to minimize the changes I would do:
```
for _, rangevar := range tests {
		tt := rangevar
		t.Run(tt.name, func(t *testing.T) {
			resp := (&tt).operation()
```

Reply:
I see, thanks @ziccardi
Will this be brought up for discussion to adjust the implementation of current unit tests within kfm?

Reply:
Yep. Added a point into the technical discussion